mysql/const.go

```diff
@@ -289,6 +289,11 @@ const (
 	MaxDoublePrecisionLength = 53
 )
 
+const (
+	SumPrecisionIncrement = 22
+	AVGPrecisionIncrement = 4
+)
+
```

Comment:
I don't think they should be added in the parser. They are relevant to the executor and it's better to be added in the tidb repo.

Reply:
ok